runtime/oti/VMHelpers.hpp

```diff
@@ -1133,7 +1133,7 @@ class VM_VMHelpers
 	 * @param[in] currentThread the current J9VMThread
 	 */
 	static VMINLINE void
-	beforeJNICall(J9VMThread *currentThread)
+	beforeJNICall(J9VMThread *currentThread, J9Method* method)
```

Comment:
Can we update the documentation for this newly added parameter?

Reply:
Added the new param to documentation.